src/Core/Helpers/AbstractConfigHelper.php

```diff
@@ -68,4 +68,9 @@ abstract class AbstractConfigHelper {
    */
   abstract public function hasBooted(): bool;
 
+  /**
+   * @return string
+   */
```

Comment:
Don't forget to add docblock for this
(And the rest of the abstract functions in this contract, if you have time)